src/nifake/unit_tests/test_converters.py

```diff
@@ -232,11 +232,66 @@ def test_repeated_capabilies_slice_prefix():
 
 
-def test_repeated_capabilies_without_prefix():
+def test_repeated_capabilities_without_prefix():
     test_result = _converters.convert_repeated_capabilities_without_prefix((slice(0, 1), '2', [4, '5-6'], '7-9', '11:14', '16, 17'))
     assert test_result == '0,2,4,5,6,7,8,9,11,12,13,14,16,17'
 
 
+def test_repeated_capabilities_string_resource_name():
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix('Dev1')
+    assert test_result_list == 'Dev1'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix('Dev1,Dev2')
+    assert test_result_list == 'Dev1,Dev2'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix('Dev1/0')
+    assert test_result_list == 'Dev1/0'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix('Dev1/0:1, Dev2/0-1')
+    assert test_result_list == 'Dev1/0:1,Dev2/0-1'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix('Dev1, Dev2/0-2')
+    assert test_result_list == 'Dev1,Dev2/0-2'
+
+
+def test_repeated_capabilities_list_resource_name():
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(['Dev1'])
+    assert test_result_list == 'Dev1'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(['Dev1', 'Dev2'])
+    assert test_result_list == 'Dev1,Dev2'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(['Dev1/0'])
+    assert test_result_list == 'Dev1/0'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(['Dev1/0:1', 'Dev2/0-1'])
+    assert test_result_list == 'Dev1/0:1,Dev2/0-1'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(['Dev1', 'Dev2/0-2'])
+    assert test_result_list == 'Dev1,Dev2/0-2'
+
+
+def test_repeated_capabilities_tuple_resource_name():
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(('Dev1'))
+    assert test_result_list == 'Dev1'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(('Dev1', 'Dev2'))
+    assert test_result_list == 'Dev1,Dev2'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(('Dev1/0'))
+    assert test_result_list == 'Dev1/0'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(('Dev1/0:1', 'Dev2/0-1'))
+    assert test_result_list == 'Dev1/0:1,Dev2/0-1'
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(('Dev1', 'Dev2/0-2'))
+    assert test_result_list == 'Dev1,Dev2/0-2'
+
+
+def test_repeated_capabilities_mixed_resource_name():
+    test_result_list = _converters.convert_repeated_capabilities_without_prefix(['Dev1', ('Dev2/0', 'Dev2/1'), ['Dev3/0:1', 'Dev4/1:2'], 'Dev5/1'])
+    assert test_result_list == 'Dev1,Dev2/0,Dev2/1,Dev3/0:1,Dev4/1:2,Dev5/1'
+
+
+def test_repeated_capabilities_invalid_resource_names():
```

Comment:
`0/1:2` would be another good test case.

Reply:
added